### PR TITLE
Update tmr-meeting-recording-change.md

### PR DESCRIPTION
**Pull request by YuriySamorodov**
Adjusted description to make it more clear (tenant member vs group member)

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tmr-meeting-recording-change.md [hunk 1]
```diff
@@ -118,9 +118,9 @@ Set-CsTeamsMeetingPolicy -Identity Global -RecordingStorageMode "Stream"
...
 |1:1 call with an external call             |Caller                 |Caller’s OneDrive for Business account                        |Caller is owner and has full rights.<br /> <br />Callee has no access. Caller must share it to the Callee.|
 |1:1 call with an external call             |Callee                 |Callee’s OneDrive for Business account                        |Callee is owner and has full rights.<br /><br />Caller has no access. Callee must share it to the Caller.|
-|Group call                                 |Any member of the call |Member who clicked on Record’s OneDrive for Business account  |Member who clicked on Record has full rights. <br /><br /> Other members from the same tenant have Read rights. <br /><br /> Other members from different tenant have no rights to it.|
+|Group call                                 |Any member of the call |Group member who clicked on Record’s OneDrive for Business account  |Member who clicked on Record has full rights. <br /><br /> Other  fr from the same tenant have Read rights. <br /><br /> Other group members from different tenant have no rights to it.|
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:121]:
```suggestion
|Group call                                 |Any member of the call |Group member who clicked on Record’s OneDrive for Business account  |Member who clicked on Record has full rights. <br /><br /> Other group members from the same tenant have Read rights. <br /><br /> Other group members from different tenants have no rights to it.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**